docs/user-guide/deploying-your-docs.md

```diff
@@ -59,21 +61,33 @@ my-project/
 orgname.github.io/
 ```
 
-After making and verifying updates to your project you need to change
-directories to the `orgname.github.io` repository and call the
-`mkdocs gh-deploy` command from there:
+After making and verifying updates to the `my-project` repository, change
+directories to the `orgname.github.io` repository and call the `mkdocs
+gh-deploy` command from there:
 
 ```sh
 cd ../orgname.github.io/
-mkdocs gh-deploy --config-file ../my-project/mkdocs.yml --remote-branch master
+mkdocs gh-deploy --config-file ../my-project/mkdocs.yml --remote-branch main
 ```
 
-Note that you need to explicitly point to the `mkdocs.yml` configuration file as
-it is no longer in the current working directory. You also need to inform the
-deploy script to commit to the `master` branch. You may override the default
-with the [remote_branch] configuration setting, but if you forget to change
-directories before running the deploy script, it will commit to the `master`
-branch of your project, which you probably don't want.
+Please note that you need to explicitly point to the `mkdocs.yml` configuration
+file in the `my-project` folder, as it is now no longer located in your current
+working directory.  
+You will also need to specify the `main` branch as the branch the deploy
+script should to commit to.
```

Comment:
```suggestion
script should commit to.
```

Reply:
I thought I'd fixed this one in one of the suggestions. I guess it didn't submit.
Edit: I was right, it's already in there to be reviewed. But I forgot to click Finish your review.